src/molecule_plugins/podman/driver.py

```diff
@@ -243,3 +243,7 @@ def sanity_checks(self):
     def required_collections(self) -> dict[str, str]:
         """Return collections dict containing names and versions required."""
         return {"containers.podman": "1.7.0", "ansible.posix": "1.3.0"}
+
+    def reset(self):
+        # keep `--filter` in sync with playbooks/create.yml
+        run_command(["podman", "rm", "--force", "--filter=owner=molecule"])
```

Comment:
I've tested that the other day. It doesn't seem to be working for me. podman doesn't like this syntax:
```
$ podman ps --filter=owner=molecule
Error: owner is an invalid filter
```
Does your syntax require a very recent version of podman ? here it seems to be ``--filter=label=owner=molecule``.

Reply:
ah sorry, typo. Swear I tested it but since this call to `run_command` didn't have `check=True` I guess I missed that. Addressed in 68ac3a4